lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -892,7 +928,9 @@
         "xsavec",
         "xsaveopt",
         "clflushopt",
-        "popcnt"
+        "popcnt",
+        "smap",
+        "smep"
```

Comment:
These two versions seem they might depend on kernel being at a [version greater than 3.7 for linux](https://en.wikipedia.org/wiki/Supervisor_Mode_Access_Prevention) and fairly recent also for other platforms. Can we remove them or is there a reason why they were added?

Reply:
Just to help distinguish. I can remove.